nodebuilder/blobstream/data_root_tuple_root.go

```diff
@@ -180,15 +181,21 @@ func proveDataRootTuples(encodedDataRootTuples [][]byte, rangeStartHeight, heigh
 // end is not included in the range.
 func (s *Service) fetchEncodedDataRootTuples(ctx context.Context, start, end uint64) ([][]byte, error) {
 	encodedDataRootTuples := make([][]byte, 0, end-start)
+	headers := make([]*nodeheader.ExtendedHeader, 0)
```

Comment:
nit but you can already cap the slice here bc you know the expected size

```
make([]*nodeheader.ExtendedHeader, 0, end-start)
```

Reply:
[1183e46](https://github.com/celestiaorg/celestia-node/pull/3470/commits/1183e46a694e00b659e82fad3b7981ab20c8a957)